core/src/components/input/input.ionic.outline.scss

```diff
@@ -0,0 +1,162 @@
+@import "./input.vars";
+
+// Input Fill: Outline (Ionic Theme)
+// ----------------------------------------------------------------
+
+:host(.input-fill-outline) {
+  --border-radius: 4px;
+  --padding-start: 12px;
+  --padding-end: 12px;
+
+  min-height: 40px;
+}
+
+/**
+ * The bottom content should never have
+ * a border with the outline style.
+ */
+:host(.input-fill-outline) .input-bottom {
+  border-top: none;
+}
+
+/**
+ * Outline inputs do not have a bottom border.
+ * Instead, they have a border that wraps the
+ * input + label.
+ */
+:host(.input-fill-outline) .input-wrapper {
+  border-bottom: none;
+}
+
+:host(.input-fill-outline.input-label-placement-stacked) .label-text-wrapper,
+:host(.input-fill-outline.input-label-placement-floating) .label-text-wrapper {
+  @include transform-origin(start, top);
+
+  position: absolute;
+
+  /**
+   * Label text should not extend
```

Comment:
~ this looks to be copy and paste, we could probably remove the 40 character line wrapping since the file already extends wider than this line would be.

Reply:
True that this was copy-pasted, but it's just Liam's commenting style; this sort of thing is all over our codebase right now 😄